codecov_cli/commands/empty_upload.py

```diff
@@ -12,11 +12,13 @@
 
 
 @click.command()
+@click.option("--force/--no-force", default=False)
```

Comment:
I'll leave the comment here but know that this is a _personal_ opinion and you can ignore this. It's also veeery nit. But if it resonates with you... :P 

I saw that this pattern is also used in `label_analysis` command. But I don't particularly like it. Having a `--no-force` flag makes no sense cause it's the default anyway. And I don't like the syntax either.

I'd recommend `@click.option("--force", is_flag=True, default=False)` instead